packages/beluga/beluga.1.1.2/opam

```diff
@@ -0,0 +1,56 @@
+opam-version: "2.0"
+synopsis:
+  "Implementation of contextual modal logic for reasoning with higher-order abstract syntax"
+maintainer: ["marc-antoine.ouimet@mail.mcgill.ca"]
+authors: [
+  "Brigitte Pientka"
+  "Joshua Dunfield"
+  "Andrew Cave"
+  "Jacob Thomas Errington"
+  "Junyoung Clare Jang"
+  "Marc-Antoine Ouimet"
+]
+license: "GPL-3.0-only"
+homepage: "http://complogic.cs.mcgill.ca/beluga/"
+bug-reports: "https://github.com/Beluga-lang/Beluga/issues"
+depends: [
+  "dune" {>= "3.7"}
+  "dune-build-info" {>= "3.7.0"}
+  "extlib" {>= "1.7.9"}
+  "gen" {>= "1.0"}
+  "linenoise" {>= "1.4.0"}
+  "ocaml" {>= "4.14"}
+  "sedlex" {>= "2.5"}
+  "omd" {>= "1.3.2"}
+  "uri" {>= "4.2.0"}
+  "ocamlformat" {= "0.25.1" & with-test}
+  "yojson" {>= "2.0.2" & with-test}
+  "ounit2" {>= "2.2.6" & with-test}
+  "bisect_ppx" {>= "2.8.1" & with-test}
+  "odoc" {>= "2.2.0" & with-doc}
```

Comment:
```suggestion
  "conf-timeout" {with-test}
  "odoc" {>= "2.2.0" & with-doc}
```
Tests are calling a shell script that uses `timeout`. They fail on macos due to its absence